tests/oeedger8r/host/CMakeLists.txt

```diff
@@ -32,5 +32,12 @@ add_executable(edl_host
     ${bar_u}
 )
 
+# The tests intentionally use floats etc in size context.
+# Disable warnings.
+set_source_files_properties(
+    ${all_u}
+    PROPERTIES COMPILE_FLAGS "-Wno-float-conversion"
```

Comment:
Windows is issuing an error because of the -Wno-float-conversion flag

Reply:
Fixed.